service/controller/v13/resource/deployment/template/bastion_e2e_vm.json

```diff
@@ -0,0 +1,117 @@
+{
```

Comment:
This specific template is not ready yet. You can review it when the PR is marked as `Ready`.